src/main/groovy/org/shipkit/gradle/ReleaseConfiguration.java

```diff
@@ -69,6 +70,21 @@ public boolean isDryRun() {
         return dryRun;
     }
 
+    /**
+     * See {@link #isReleaseAllJavaModules()}}
+     */
+    public void setReleaseAllJavaModules(boolean releaseAllJavaModules) {
+        this.releaseAllJavaModules = releaseAllJavaModules;
+    }
+
+    /**
+     * org.shipkit.java-library plugin will be applied to every java subproject if this boolean
```

Comment:
Nice docs! Let's make it explicit what 'java subproject' means, something like that:

"org.shipkit.java-library plugin will be applied to every java subproject (project that applies Gradle's 'java' plugin) if this ..."